modules/client/src/client-common/interfaces/common.ts

```diff
@@ -11,6 +16,7 @@ export type NetworkDeployment = {
   adminSetup: string;
   addresslistVotingSetup: string;
   tokenVotingSetup: string;
+  publicResolver?: string;
```

Comment:
The variable should include the ENS registry, which we deploy
The resolver is defined at the registry level